docs/accessibility/index.md

```diff
@@ -87,3 +87,40 @@ const App = () => {
 	);
 };
 ```
+
+## Debug Accessibility
+
+Enact also provides ways to debug these accessibility implementations.
+
+### In sampler
+
+We provide a sampler in [UI COMPONENTS](https://enactjs.com/sampler/sandstone) that implements several UI components as [Storybook](https://storybook.js.org/).
+If click the `debug aria` tab located at the top of the page of the sampler and select `true`, the ARIA properties of each component is displayed as shown in the image below.
+
+![debug-aria in sampler](Debugaria-Sampler.png)
+
+### In Enact app
+
+This debug feature is designed to work if a class `debug aria` is given to any enact app.
+So just add `debug aria` to className prop in your enact app to see the applied ARIA properties.
+
+The following example demonstrates the usage of `debug aira` className.
+
+```js
+import ThemeDecorator from '@enact/sandstone/ThemeDecorator';
+
+const AppBase = (props) => {
+	return (
+		<div {...props}>
+			...
+		</div>
+	);
+};
+const debugAppBase = () => {
+	return (
+		<AppBase className='debug aria' />
```

Comment:
Again, this is the question. 
Don't we have any sample that condition check on debug mode only enabled?

Reply:
I remained the answer in ticket comments.